keyboard/ergodox_ez/keymaps/ben/keymap.c

```diff
@@ -16,41 +16,41 @@ const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
  * ,--------------------------------------------------.           ,--------------------------------------------------.
  * | ESC    |  1/! |  2/" |  3/£ |  4/$ |  5/% |  6/^ |           |  7/& |  8/ *|  9/( |  0/) |  -/_ | =/+  | BkSp   |
  * |--------+------+------+------+------+-------------|           |------+------+------+------+------+------+--------|
- * | Tab    |   Q  |   W  |   E  |   R  |   T  |  Y   |           |  L1  |   Y  |   U  |   I  |   O  |   P  | Del    |
+ * | Tab    |   Q  |   W  |   E  |   R  |   T  |  Y   |           |  L2  |   Y  |   U  |   I  |   O  |   P  | Del    |
```

Comment:
No particular reason. L1 toggle would be closer to a thumb, like L2 toggle on the LH.